### DISCUSSION TITLE
Plot of misorientation angle of GBs with specimen stress axis

### DISCUSSION THREAD

**Discussion by EiriniGalliopoulou** [ORTools4MTEX/ORTools#40] [Q&A] [answered]
Hello Dr.Niessen and team,

I would like to ask how could I plot the misorientation angle of GBs with respect to the stress axis of the specimen? I believe we define which is the stress axis, when we first load the .ctf file but I am not sure how to plot the mis angle of GBs with respect to that. Any help would be much appreciated!

Eirini

**Comment by AzdiarGazder** [comment, marked as answer]
Hi @EiriniGalliopoulou,

It is only possible to plot the **misorientation axis** (and not gB misorientation angles) with respect to a particular specimen coordinate system.

To do this, please refer to the following weblink:
https://mtex-toolbox.github.io/HomepageOld/files/doc/GrainMisorientationAnalysis.html

The relevant section begins with the sentence:
"The misorientation axis with respect to the specimen coordinate system can unfortunately not be computed from the misorientation alone. Therefore, we require the pair consisting of grain mean orientation and the orientation of the pixel."

Hope this helps.

Warm regards,
Azdi
